src/OneExplorer/ConfigObject.ts

```diff
@@ -19,6 +19,191 @@ import * as ini from 'ini';
 import * as path from 'path';
 import * as vscode from 'vscode';
 
+/**
+ * 'Artifact' means includes
+ *   - existing files to run ONE config (base model)
+ *   - result files after running ONE config (derived models, )
+ */
+interface Artifact {
+  /**
+   * An artifact's type
+   */
+  type: string;
+
+  /**
+   * A file extension
+   */
+  ext: string;
+
+  /**
+   * A full path in file system
+   */
+  path?: string
+}
+
+/**
+ * 'ArtifactLocator' is to find artifact's absolute path
+ */
+export class ArtifactLocator {
+  /**
+   * The section of ini to find the targetted file
+   * If not given, locator searches the whole section
+   */
+  section?: string;
+
+  /**
+   * The key inside section to find the targetted file
+   * If not given, locator searches the whole key
+   */
+  key?: string;
+
+  /**
+   * (optional)
+   * A search function to find the file in a space-seperated word sequence
+   *
+   * If undefined, locator searches with 'ext' within the given ['section']['key'] field.
+   * Refer to `this.searchEndsWithExt`
+   */
+  search: (arg: string) => string[];
+
+  /**
+   * A target artifact
+   */
+  artifact: Artifact;
+
+  /**
+   * A default search function
+   * @param content [this.section][this.key] value
+   * @returns string file name array
+   */
+  private searchEndsWithExt = (content: string): string[] => {
+    // Don't remove this. It's to prevent an 'content.cplit is not a function' error.
```

Comment:
Could you describe this error in detail? Hmm. The code doesn't seem to have the problem.

Reply:
I cannot reproduce this.

Reply:
If you remove the line and run `npm run unittest`, you can see the error.

BTW, there is a typo
'content.cplit' -> 'content.split'

Reply:
Very interesting!

I can reproduce
```
###True
###True
###/dummy/dummy/../../model.tflite
###/dummy/dummy/../../model.circle
###/model.circle
###/dummy/dummy/../..//model.q8.circle
###true
```

The contents' values are also `True` or `true`?

Anyway, the code seems okay on vscode npm watch. However, running unittests causes this error.

I guess
- On ts, the content is assumed as string type
- On js, content cannot be sure as string type because it is on js world
- the content is boolean type or functioin about boolean, I guess
- So, content(`not string type`) cannot have split function

I think that `content = content + ''` can be good option because it can make it sure the type is string.